sources/packages/forms/src/form-definitions/studentfinancialinformationappeal.json

```diff
@@ -1549,6 +1561,1188 @@
+                    "showCharCount": false,
+                    "showWordCount": false,
+                    "allowMultipleMasks": false,
+                    "addons": [],
+                    "inputType": "radio",
+                    "fieldSet": false,
+                    "id": "ee3su14",
+                    "defaultValue": "",
+                    "lockKey": true
+                },
+                {
+                    "title": "Decrease in partner income panel",
+                    "theme": "default",
+                    "breadcrumb": "default",
+                    "breadcrumbClickable": false,
+                    "buttonSettings": {
+                        "previous": false,
+                        "cancel": false,
+                        "next": false
+                    },
+                    "scrollToTop": false,
+                    "tooltip": "",
+                    "customClass": "",
+                    "collapsible": false,
+                    "hidden": false,
+                    "hideLabel": true,
+                    "disabled": false,
+                    "modalEdit": false,
+                    "key": "decreaseInPartnerIncomePanel",
+                    "tags": [],
+                    "properties": {},
+                    "customConditional": "",
+                    "conditional": {
+                        "json": "",
+                        "show": "true",
+                        "when": "hasCurrentYearPartnerIncome",
+                        "eq": "yes"
+                    },
+                    "nextPage": "",
+                    "logic": [],
+                    "attributes": {},
+                    "overlay": {
+                        "style": "",
+                        "page": "",
+                        "left": "",
+                        "top": "",
+                        "width": "",
+                        "height": ""
+                    },
+                    "type": "panel",
+                    "label": "Decrease in Income. Panel",
+                    "tabindex": "",
+                    "input": false,
+                    "tableView": false,
+                    "allowPrevious": false,
+                    "components": [
+                        {
+                            "input": true,
+                            "tableView": false,
+                            "key": "currentYearPartnerIncomeApplicationException",
+                            "label": "Application Exception - Current Year Income",
+                            "protected": false,
+                            "unique": false,
+                            "persistent": true,
+                            "type": "hidden",
+                            "tags": [],
+                            "conditional": {
+                                "show": "",
+                                "when": null,
+                                "eq": ""
+                            },
```

Comment:
Please do not copy over the exceptions to the appeal. Appeals (Change requests does not have exception process).